Makefile

```diff
@@ -0,0 +1,27 @@
+HARDHAT_BASE_COMMAND=npx hardhat
```

Comment:
A `Makefile`!  I didn't know JS or Solidity projects even allowed makefiles!  

(Plus file doesn't end with a NL.)

Reply:
Yeah, I've been using Makefiles for a wide range of personal projects, including Go, Java, React, Solidity, I. think it is allowed for a lot more.

Reply:
(The "allowed" was a joke ... you're going to have to watch out for my out-of-control "sense of humor")